### PR TITLE
Add program name "Nebula" to the /nasa homepage.

### PR DESCRIPTION
**Pull request by NPDebs**
This PR resolves issue #762.
When the `/nasa` subsite was created, the program had not been branded.
Here, the program name (Nebula) has been highlighted in select parts of the homepage.

**Changes made:**
1. Mention program name in _/nasa_ index page.
2. Rename _Open Science Training_ navbar item in header. 
3. Change the workstream name from NASA-TOPS to Nebula.

**Screenshots:**

add_nebula
add_nebula1

Live [preview here](https://deploy-preview-763--ols-bebatut.netlify.app/open-science-training)...

Thank you for reviewing!

### CODE REVIEW

**Comment by bebatut** [comment]
The changes look good.
I have just a question. As Open Seed subsite has everything within folder named `openseeds`, should we rename every folder `nebula` instead of `nasa`?

**Comment by NPDebs** [comment]
> I have just a question. As Open Seed subsite has everything within folder named `openseeds`, should we rename every folder `nebula` instead of `nasa`?


🤔 Yeah, that would be the logical thing to do. Shall I go ahead and push that change?

**Comment by bebatut** [comment]
> Shall I go ahead and push that change?

I think so. In worst case, we can revert it
Keep in mind to add a `redirect_from` to every page renamed

**Comment by NPDebs** [comment]
Fortunately, we currently don't have that many pages within the /nebula family.

File: _includes/default-header.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
             <a class="navbar-link" href="{% link open-science-training.md %}">Open Science Training</a>
             <div class="navbar-dropdown">
               <a class="navbar-item" href="{% link openseeds/index.md %}">Open Seeds</a>
-              <a class="navbar-item" href="{% link nasa/index.md %}">NASA-OLS TOPS Open Core</a>
+              <a class="navbar-item" href="{% link nasa/index.md %}">Nebula</a>
```
Comment (bebatut) [inline _includes/default-header.html:25]: You probably need to update the link here

File: nasa/index.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ photos:
```
Comment (bebatut) [inline nasa/index.md:13]: You should probably remove this file

Comment (NPDebs) [reply nasa/index.md:15]: Thanks! I deleted the `nasa/index.md` but didn't commit it. So, it didn't reflect.

I think that's fixed now...

**Comment by bebatut** [comment]
Just need to be careful that the link is also updated in all branding material

**Comment by NPDebs** [comment]
> Just need to be careful that the link is also updated in all branding material

Sure...

**Comment by yochannah** [comment]
The old link will work though, right, because of the redirect?

**Review by bebatut — COMMENTED** [review]
I think it is ready for merging. @yochannah should we? or wait for @iramosp feedback?

**Review by yochannah — APPROVED** [review]

**Comment by yochannah** [comment]
I've merged - @iramosp please PR or raise an issue if you have more feedback!

**Pull request merged**